### PR TITLE
CASMINST-6617: Update kubectl example output

### PR DESCRIPTION
**Pull request by mharding-hpe**
Backport of https://github.com/Cray-HPE/docs-csm/pull/4161, but with the CSM 1.5 versions

### CODE REVIEW

File: install/deploy_non-compute_nodes.md [hunk 1]
```diff
@@ -229,12 +229,12 @@ for all nodes, the Ceph storage will have been initialized and the Kubernetes cl
...
+        ncn-m002   Ready    control-plane,master   7m39s   v1.22.13   10.252.1.5    <none>        SUSE Linux Enterprise High Performance Computing 15 SP4   5.14.21-150400.24.66-default   containerd://1.5.16
+        ncn-m003   Ready    control-plane,master   7m16s   v1.22.13   10.252.1.6    <none>        SUSE Linux Enterprise High Performance Computing 15 SP4   5.14.21-150400.24.66-default   containerd://1.5.16
+        ncn-w001   Ready    <none>                 7m16s   v1.22.13   10.252.1.7    <none>        SUSE Linux Enterprise High Performance Computing 15 SP4   5.14.21-150400.24.66-default   containerd://1.5.16
+        ncn-w002   Ready    <none>                 7m18s   v1.22.13   10.252.1.8    <none>        SUSE Linux Enterprise High Performance Computing 15 SP4   5.14.21-150400.24.66-default   containerd://1.5.16
+        ncn-w003   Ready    <none>                 7m16s   v1.22.13   10.252.1.9    <none>        SUSE Linux Enterprise High Performance Computing 15 SP4   5.14.21-150400.24.66-default   containerd://1.5.16
```
Comment (rustydb) [inline install/deploy_non-compute_nodes.md]: Do we want this to say SP5 instead?

Comment (mharding-hpe) [reply install/deploy_non-compute_nodes.md]: I just grabbed it off a system that claimed to be CSM 1.5.. I just want it to say whatever CSM 1.5 systems are gonna say. :)

**Review by gbaker-hpe — APPROVED** [review]

File: install/deploy_non-compute_nodes.md (hunk 1 above)
Comment (gbaker-hpe) [reply install/deploy_non-compute_nodes.md]: It needs to be SP5...

Comment (mharding-hpe) [reply install/deploy_non-compute_nodes.md]:
```suggestion
        ncn-m002   Ready    control-plane,master   7m39s   v1.22.13   10.252.1.5    <none>        SUSE Linux Enterprise High Performance Computing 15 SP5   5.14.21-150500.55.12-default   containerd://1.5.16
        ncn-m003   Ready    control-plane,master   7m16s   v1.22.13   10.252.1.6    <none>        SUSE Linux Enterprise High Performance Computing 15 SP5   5.14.21-150500.55.12-default   containerd://1.5.16
        ncn-w001   Ready    <none>                 7m16s   v1.22.13   10.252.1.7    <none>        SUSE Linux Enterprise High Performance Computing 15 SP5   5.14.21-150500.55.12-default   containerd://1.5.16
        ncn-w002   Ready    <none>                 7m18s   v1.22.13   10.252.1.8    <none>        SUSE Linux Enterprise High Performance Computing 15 SP5   5.14.21-150500.55.12-default   containerd://1.5.16
        ncn-w003   Ready    <none>                 7m16s   v1.22.13   10.252.1.9    <none>        SUSE Linux Enterprise High Performance Computing 15 SP5   5.14.21-150500.55.12-default   containerd://1.5.16
```

**Review by rustydb — APPROVED** [review]

**Pull request merged**